src/test/java/io/strimzi/HealthCheckIT.java

```diff
@@ -7,7 +11,7 @@
 import static org.hamcrest.CoreMatchers.is;
 
 @QuarkusTest
-public class HealthCheckResourceTest {
+public class HealthCheckIT {
```

Comment:
I thought we wanted to keep this as regular unit test? I'm not entirely sure I understand why we need this and HealthCheckTest.

Reply:
I created both. Although simple, I still think the original one should be classified as integration test as it requires to start the Quarkus runtime, which is an overkill for a unit test.